### DISCUSSION TITLE
Alternativas ao Google Analytics

### DISCUSSION THREAD

**Discussion by kazzkiq** [frontendbr/forum#1376] [Perguntas e Respostas]
Fala pessoal,

Ultimamente eu venho tentando me desprender ao máximo dos serviços/produtos Google, e isso inclui também o Google Analytics.

Pesquisando alternativas ao Analytics, eu percebi que a concorrência é quase inexistente, principalmente de produtos open-source.

Então gostaria de levantar essa bola aqui no forum e perguntar se **vocês conhecem alternativas (de preferência open-source) ao Analytics**. Lembrando que não precisa ter todas as features, apenas o básico (visitas/pageviews) já seria suficiente. Alguma indicação?

**Comment by luizbills** [comment]
Recentemente vi o https://usefathom.com/

**Comment by vhfmag** [comment]
Eu uso o [Matomo](https://matomo.org/) em meu blog, ele é bem completo (tem até um post explicando [porque ele é uma alternativa viável ao Google Analytics 360](https://matomo.org/blog/2018/12/why-matomo-is-a-serious-alternative-to-google-analytics-360/)) e open-source, o que significa você pode rodar a sua própria instância e os dados dos seus usuários nunca vão pra servidores de terceiros.

Já ouvi falar também do [Open Web Analytics](http://www.openwebanalytics.com/), mas nunca usei e não posso falar sobre funcionalidades.

**Comment by vhfmag** [comment]
@luizbills esse Fathom parece muito interessante!

![image](https://user-images.githubusercontent.com/357835/53170885-ffe90e00-35bf-11e9-82c8-7d03084dfff4.png)

**Comment by felipefialho** [comment]
Recomendo dar uma olhada no Amplitude.

**Comment by valterbarros** [comment]
Em relação ao Amplitude alguém poderia me informar os valores mais ou menos? Lá na empresa estão usando Yandex parece atender a necessidade mas é sempre bom ter opções.

**Comment by felipefialho** [comment]
@valterbarros Eles tem um [pacote free](https://amplitude.com/pricing?ref=nav), vale pra testar as funcionalidades

**Comment by lcquadros** [comment]
Matomo é uma boa opção pata evitar se prender demais ao google analytics.

O GA esconde informações essenciais para outros produtos (adwords) e isso
por si já é um ponto negativo.

Em seg, 25 de fev de 2019 10:53, Felipe Fialho <notifications@github.com
escreveu:

**Comment by vicainelli** [comment]
Eu uso bastante o Mixpanel e o Heap.
Mas acredito que tudo depende de qual é a  necessidade, se é tracking de usuários, números de pageviews.

- https://mixpanel.com
- https://heapanalytics.com/

**Comment by marcosalberto** [comment]
Tem o Chartbeat, ele tem foco em Tempo Real.
